virttest/bootstrap.py

```diff
@@ -300,6 +301,79 @@ def create_guest_os_cfg(t_type):
               guest_os_cfg_path)
 
 
+def host_os_get_distro_name(options, detected, compat=False):
+    """
+    :param compat: If set, name is returned as uppercase or capitalized
+    """
+    if options.vt_host_distro_name:
+        return options.vt_host_distro_name
+    if compat:
+        if detected.name == 'rhel':
+            return 'RHEL'
+        elif detected.name == 'fedora':
+            return 'Fedora'
+    return detected.name
+
+
+def host_os_get_distro_version(options, detected, compat=False):
+    """
+    :param compat: If set, distro version with an "m" (as in major) prefix
+    """
+    if options.vt_host_distro_version:
+        version = options.vt_host_distro_version
+    else:
+        version = detected.version
+    if compat:
+        return "m%s" % version
+    else:
+        return version
+
+
+def host_os_get_distro_release(options, detected, compat=False):
+    """
+    :param compat: If set, distro version with an "u" (as in update) prefix
+    """
+    if options.vt_host_distro_release:
+        release = options.vt_host_distro_release
+    else:
+        release = detected.release
+    if compat:
+        return "u%s" % release
+    else:
+        return release
+
+
+def create_host_os_cfg(options):
+    compat = options.vt_host_distro_compat
+    host_os_cfg_path = data_dir.get_backend_cfg_path(options.vt_type, 'host-os.cfg')
+    with open(host_os_cfg_path, 'w') as cfg:
+        detected = distro.detect()
+        name = host_os_get_distro_name(options, detected, compat)
+        version_raw = host_os_get_distro_version(options, detected, False)
+        version = host_os_get_distro_version(options, detected, compat)
+        release_raw = host_os_get_distro_release(options, detected, False)
+        release = host_os_get_distro_release(options, detected, compat)
+        cfg.write("variants:\n")
+        cfg.write("    - @Host_%s:\n" % name)
+        cfg.write("        host_kernel_ver_str = Host_%s\n" % name)
```

Comment:
might be useful in case of later use in test, but the same can get through distro_detect if not override with commandline param, can be there.